plugin.video.dailymotion_com/default.py

```diff
@@ -62,42 +67,50 @@ def strip_tags(html):
-    addDir(translation(30003), urlMain+"/videos?fields=id,thumbnail_large_url,title,views_last_hour&availability=1&live_onair=1&sort=visited-month&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listLive', "")
-    addDir(translation(30006), "", 'listChannels', "")
-    addDir(translation(30007), urlMain+"/users?fields=username,avatar_large_url,videos_total,views_total&sort=popular&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listUsers', "")
-    addDir(translation(30002), "", 'search', "")
-    addDir(translation(30002)+" "+translation(30003), "", 'livesearch', "")
-    addDir(translation(30002)+" "+translation(30007), "", 'usersearch', "")
+    addDir(translation(30025), urlMain+"/videos?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&list=what-to-watch&no_live=1&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listVideos', _ipath + "what_to_watch.png")
+    addDir(translation(30015), urlMain+"/videos?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&sort=trending&no_live=1&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listVideos', _ipath + "trending.png")
+    addDir(translation(30016), urlMain+"/videos?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&featured=1&no_live=1&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listVideos', _ipath + "featured.png")
+    addDir(translation(30003), urlMain+"/videos?fields=id,thumbnail_large_url,title,views_last_hour&availability=1&live_onair=1&sort=visited-month&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listLive', _ipath + "live.png")
+    addDir(translation(30006), "", 'listChannels', _ipath + "channels.png")
+    addDir(translation(30007), urlMain+"/users?fields=username,avatar_large_url,videos_total,views_total&sort=popular&limit="+itemsPerPage+"&family_filter="+familyFilter+"&localization="+language+"&page=1", 'listUsers', _ipath + "users.png")
+    addDir(translation(30002), "", 'search', _ipath + "search.png")
+    addDir(translation(30002) + " " + translation(30003), "", 'livesearch', _ipath + "search_live.png")
+    addDir(translation(30002) + " " + translation(30007), "", 'usersearch', _ipath + "search_users.png")
     if dmUser:
-        addDir(translation(30034), "", "personalMain", "")
+        addDir(translation(30034), "", "personalMain", _ipath + "my_stuff.png")
     else:
-        addFavDir(translation(30024), "", "favouriteUsers", "")
+        addFavDir(translation(30024), "", "favouriteUsers", _ipath + "favourite_users.png")
+    xbmcplugin.setContent(pluginhandle, "addons")
     xbmcplugin.endOfDirectory(pluginhandle)
+    xbmc.executebuiltin('Container.SetViewMode(500)')
```

Comment:
This isn't allowed.

Reply:
Corrected it to use only if user setup the relevant modes in settings